Directory.Build.targets

```diff
@@ -22,14 +22,13 @@
   <ItemGroup Condition="'$(IsTestProject)' == 'true'">
     <Compile Include="$(CommonPath)tests\TestUtilities\System\AssertExtensions.cs" Link="Common\System\AssertExtensions.cs" />
     <Compile Include="$(CommonPath)tests\TestUtilities\System\PlatformDetection.cs" Link="Common\System\PlatformDetection.cs" />
-    <Compile Condition="'$(OS)' != 'Windows_NT'" Include="$(CommonPath)tests\TestUtilities\System\PlatformDetection.Unix.cs" Link="Common\System\PlatformDetection.Unix.cs" />
-    <Compile Condition="'$(OS)' == 'Windows_NT'" Include="$(CommonPath)tests\TestUtilities\System\PlatformDetection.Windows.cs" Link="Common\System\PlatformDetection.Windows.cs" />
+    <Compile Condition="'$(OS)' == 'Windows_NT'" Include="$(CommonPath)src\Interop\Interop.Libraries.cs" Link="Common\Interop\Interop.Libraries.cs" />
+    <Compile Condition="'$(OS)' == 'Windows_NT'" Include="$(CommonPath)src\Interop\NtDll\Interop.RtlGetVersion.cs" Link="Common\Interop\NtDll\Interop.RtlGetVersion.cs" />
```

Comment:
These files should get imported for all host OSs, not just Windows.

Reply:
Hmm but they're Windows only. Why do we want to import them in Unix?

Reply:
That's what we do in runtime as well: https://github.com/dotnet/runtime/blob/f7937de8d3fc52e099d126e0085a679d30b285a0/src/libraries/Common/tests/TestUtilities/TestUtilities.csproj#L64C1-L65

We only have a single platform agnostic build for TestUtilities.

Reply:
See it like this: When you compile the test library on Linux but then execute it on Windows, you want runtime detection to work (which requires these files). Same vice-versa.